src/js/controller/controller.js

```diff
@@ -263,9 +265,11 @@ define([
 
                 switch (typeof item) {
                     case 'string':
+                        _this.trigger(events.JWPLAYER_LOAD_PLAYLIST, {});
                         _loadPlaylist(item);
                         break;
                     case 'object':
+                        _this.trigger(events.JWPLAYER_LOAD_PLAYLIST, {});
```

Comment:
Either we make a 'load' event that fires for all three cases, or we make a 'loadPlaylist" and a 'beforeSetPlaylist' event.